viewing_party/party.py

```diff
@@ -1,23 +1,173 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+
+    movie ={}
+    movie["title"] = title
+    movie["genre"] = genre
+    movie["rating"] = rating

```

Comment:
This works well! As an alternative, can you consider how you might make this dictionary in a single statement all at once?